release/RELEASE.md

```diff
@@ -0,0 +1,105 @@
+  commit
+  branch v4_9-STABLE
+  commit tag:"v4.9.0rc1"
+  commit tag:"v4.9.0rc2"
+  commit tag:"v4.9.0"
+  commit tag:"d4.9.0-0"
+  commit tag:"d4.9.0"
+  checkout development
+  commit
+  commit
+  branch v5_0-STABLE
+  commit
+  commit tag:"v5.0.0rc1"
+  commit tag:"v5.0.0rc2"
+  commit tag:"v5.0.0"
+  commit
+  commit tag:"d5.0.0-0"
+  commit tag:"d5.0.0"
+  checkout development
+  commit
+```
+16. Build team builds final code and lock package versions 
+17. Preprod testing/validation
+18. Prod rollout to access portal
+19. Documentation release to access portal via the doc tag for this release.  If needed, additional documentation tweaks can be done against the `v4_9-STABLE` branch and the doc tag can be moved (though see "Making changes to back branches", step 6 for guidance).
+20. Release notes and release blog article/customer promotion.
+21. Announce release to various channels: #releases on Slack, marketing email, blog, company-wide email, etc.
+
+## Making changes to back branches
+
+A "back branch" is considered any branch which belongs to an already-released version of pgMonitor, so effectively anything which is not currently in development.  Note: at this time back branches are currently only supported for HA 4.8.0 and later.
+
+If a bug fix in the code or documentation for a given release is required, you should follow the following procedures:
+
+1. Checkout a new branch based on the appropriate upstream branch.  For instance if you were patching pgMonitor release 4.9.0, you would use `upstream/v4.9.0-STABLE` as the base.
+2. Make your local changes and test appropriately.  Depending on which version you are adjusting there may or may not be automated testing support.
+3. Create a new PR with the base of the target branch. Go through the normal PR review/acceptance process.  Merge the PR into the appropriate upstream branch.
+4. Checkout the upstream primary branch.
+5. If the changes require new packages to be built you will need to tag a new release:
+   - for 4.8.0 and higher you will need to tag a new patch release with semantic versioning; i.e.: `v4.9.1`
+6. If the changes involve any documentation updates, you will need to:
+   a. tag a new specific doc release revision; i.e., `d4.9.0-N`. This corresponds to an immutable "docs have been released" version and allows us to track.
+   b. ensure that the docs changes are what are expected; `git diff d4.9.0-2 d4.9.0-3 -- hugo` (using the appropriate current release and previous release).  If you notice something wrong at this point and you have not pushed this tag you can correct locally and re-tag with the same release version.  If this tag has already been pushed, correct and increment the docs release number, repeating step 5.
+   c. update this release's main documentation build tag: i.e., `git tag -f d4.9.0 d4.9.0-3` to force the current 4.9.0 docs to be updated with this contents of this tag.  This is required for the build team's website generation.
```

Comment:
I think this step is no longer required. And if it was required, we'd not be updating `d4.9.0`, we'd be updating `d4.9`, right? (Since we'd be updating the 4.9 docs generally, not docs for the specific patch release. Or am I missing something here?)

Reply:
So I'd say whack C entirely and move current D to C.